shamedpets/app/views/offenses/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_for @offense do |f| %>
+
+  <% if @offense.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluralize(@offense.errors.count, "error") %>
+    prohibited
```

Comment:
maybe change prohibited to something more descriptive "The data you entered had the following errors"